src/coreComponents/mesh/CMakeLists.txt

```diff
@@ -105,7 +105,7 @@ set( mesh_sources
      utilities/ComputationalGeometry.cpp
      )
 
-set( dependencyList schema dataRepository constitutive metis )
+set( dependencyList schema dataRepository constitutive finiteElement metis )
```

Comment:
I would better expect `finiteElement` to depend on `mesh` than the opposite?
Maybe this means that we need to extract some `computationalGeometry` cmake target?

Reply:
The dependency on `finiteElement` for `computationalGeometry` is a temporary solution to have access to isoparametric Jacobian transformation matrix/mapping from the parent space to the physical space. Ultimately such capabilities should be provided through a numerical integration library. I am happy to volunteer for implementing it, but this should be an independent effort. @rrsettgast @klevzoff thoughts?